keras_cv/ops/__init__.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from keras_cv.keypoint.converters import convert_format
-from keras_cv.keypoint.formats import REL_XY
-from keras_cv.keypoint.formats import XY
-from keras_cv.keypoint.utils import filter_out_of_image
+from keras_cv.ops.keypoint_converters import convert_format
```

Comment:
Should we keep a sub-package as ops.key_point.xxx? The current API like `keras_cv.ops.convert_format` lost the context of keypoints.

Reply:
+1 -- or perhaps the method should have context in-name (e.g. convert_keypoint_format)

Reply:
"convert_keypoint_format" sounds good to me
We also need to consolidate "REL_XY" in keypoint and "REL_XY" in bbox, maybe next PR then...